### PR TITLE
Inprecise optional variables

### PR DESCRIPTION
**Pull request by jogobeny**
Hi, Trading212 API docs are not precise in some optional variables, so I've made a few changes based on my API requests.

### CODE REVIEW

File: trading212/models.py [hunk 1]
```diff
@@ -183,32 +185,32 @@ class Tax(BaseModel):
...
-    fillType: str
-    filledQuantity: float
-    filledValue: float
+    fillCost: float | None
+    fillId: int | None
+    fillPrice: float | None
+    fillResult: float | None
+    fillType: str | None
+    filledQuantity: float | None
+    filledValue: float | None
```
Comment (jcoelho93) [inline trading212/models.py]: @jogobeny Thanks for the contribution. Would you mind using the `Optional` field type instead?

```suggestion
    fillCost: Optional[float] = None
    fillId: Optional[int] = None
    fillPrice: Optional[float] = None
    fillResult: Optional[float] = None
    fillType: Optional[str] = None
    filledQuantity: Optional[float] = None
    filledValue: Optional[float] = None
```

File: trading212/models.py [hunk 2]
```diff
@@ -142,7 +144,7 @@ class Order(BaseModel):
 
 
 class AccountCash(BaseModel):
-    blocked: float
+    blocked: float | None
```
Comment (jcoelho93) [inline trading212/models.py]:
```suggestion
    blocked: Optional[float] = None
```

File: trading212/trading212.py [hunk 3]
```diff
@@ -353,7 +353,7 @@ def fetch_specific_position(self, ticker: str) -> Position:
 
         return Position(**response)
 
-    def historical_order_data(self, cursor: int, ticker: str, limit: int) -> HistoricalOrderData:
+    def historical_order_data(self, cursor: int | None = None, ticker: str | None = None, limit: int = None) -> HistoricalOrderData:
```
Comment (jcoelho93) [inline trading212/trading212.py]:
```suggestion
    def historical_order_data(self, cursor: Optional[int] = None, ticker: Optional[str] = None, limit: Optiona[int] = None) -> HistoricalOrderData:
```

**Review by jcoelho93 — CHANGES_REQUESTED** [review]
Approved pending using Optional type

**Comment by jogobeny** [comment]
It may be worth releasing a new version of the pip package because those requests were not working because of Pydantic validating.

**Review by jcoelho93 — APPROVED** [review]

**Comment by jcoelho93** [comment]
Fixed in `2.0.4`

**Pull request merged**